auto_submit/lib/validations/ci_successful.dart

```diff
@@ -152,4 +172,16 @@ class CiSuccessful extends Validation {
 
     return allSuccess;
   }
+
+  List<String> _checkRunListToStringList(List<github.CheckRun> checkRuns) {
+    List<String> checkRunStrings = [];
+    for (int i = 0; i < checkRuns.length; i++) {
```

Comment:
You can avoid this. Dart will do this for you when interpolating a List<CheckRun> to a string

Reply:
So we don't get this because when we are currently trying to do just that. Since CheckRun does not implement toString we get the following:

```
severity: "INFO"
textPayload: "auto_submit: Validating name: flutter, checks: [Instance of 'CheckRun', ..., Instance of 'CheckRun']"
```

Reply:
It doesn't or isn't (for some reason) interpolating correctly here.

Reply:
Ah, package:github implements toString, but we have technical debt with our own CheckRun in cocoon as well. That needs to implement toString as well

Reply:
Haha yeah I just saw that.